spanner/changestreams/src/test/java/com/example/spanner/changestreams/ChangeStreamSampleIT.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner.changestreams;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Tests for ChangeStreamSample.
+ */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class ChangeStreamSampleIT {
+  private static String instanceId = System.getProperty("spanner.test.instance");
+  private static String databaseId = System.getProperty("spanner.test.database");
+  private static String prefix = System.getProperty("spanner.test.prefix");
+
+  private ByteArrayOutputStream bout;
+  private PrintStream stdOut = System.out;
+  private PrintStream out;
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+  }
+
+  @After
+  public void tearDown() {
+    try {
+      bout.close();
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+
+    System.setOut(stdOut);
+  }
+
+  @Test
+  public void testChangeStreamSample() {
+    assertNotNull(instanceId);
```

Comment:
IT tests seem to be failing here

Reply:
We need to set system properties in the POM file to specify the instance, database and prefix (per above). An example of how to do this can be seen here: https://github.com/googleapis/java-spanner/blob/main/samples/snippets/pom.xml#L149

We need to find out which projects / instance / database can be used here.

Reply:
The project seems to be `java-docs-samples-testing` as specified here: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.kokoro/tests/run_tests.sh#L56

I don't have access to it, so I don't know which spanner instance / database we can use. 

@averikitsch do you know if there is a Spanner instance / database that we can use in this project for integration testing?
@haikuo-google we will need to allowlist this project for change streams